src/openrct2/ride/gentle/CarRide.cpp

```diff
@@ -433,16 +433,16 @@ static void PaintCarRideTrackLeftQuarterTurn1Tile(
     switch (direction)
     {
         case 0:
-            PaintAddImageAsParent(session, imageId, { 6, 0, height }, { 26, 24, 1 }, { 6, 2, height });
+            PaintAddImageAsParent(session, imageId, { 6, 0, height }, { { 6, 2, height }, { 26, 24, 1 } });
             break;
         case 1:
-            PaintAddImageAsParent(session, imageId, { 0, 0, height }, { 26, 26, 1 });
+            PaintAddImageAsParent(session, imageId, { 0, 0, height }, { { 0, 0, 0 }, { 26, 26, 1 } });
```

Comment:
If no bound box offsets are specified, the regular offset is used.
```suggestion
            PaintAddImageAsParent(session, imageId, { 0, 0, height }, { { 0, 0, height }, { 26, 26, 1 } });
```